src/storage/Backend.h

```diff
@@ -235,7 +237,8 @@ class Backend : public zeek::Obj {
     TypePtr val_type;
     RecordValPtr backend_options;
 
-    std::string tag;
+    zeek::Tag tag;
+    std::string tag_str;
```

Comment:
Is it worth keeping the string version in addition to the tag?

Reply:
The string version is because if I use `Tag::AsString()` in `Tag()`, and then try to return the `.c_str` output, the compiler warns that it's returning the address of a temporary.